### PR TITLE
Changes to create stack

### PR DESCRIPTION
**Pull request by knowuh**
Updates to the `create-stack` script, documentation, and sample files.

This work was done while documenting support for setting up LARA/Portal pairs in the QA environment.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
+* optionally one of the following source Param types:
+    * `StackParams` — The file under `stack-params` to use for default params
+    * `OriginalStack` — The name of a running stack to copy params from
+    * If no params are specified, you will be prompted to choose an existing
+    parameter set from `stack-params/*.yml`
+* `ParameterModifications` — Values to replace from the stack-params.
+
+## .gitignore
+Because you will be creating config files that might have passwords in them,
+git will ginore `.yml` files in the `configs` and `stack-params` folders. Sample
```
Comment (kswenson) [inline scripts/README.md:29]:
```suggestion
git will ignore `.yml` files in the `configs` and `stack-params` folders. Sample
```

File: scripts/README.md [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+
+## Create a new running stack from a configuration file
+If you choose the `Create stack` action, you will be shown a list of config
+files from the `configs` directory. The script will fail if you have no
+configurations in this folder.
+
+## Config File format
+Files in the `configs` directory should specify the following parameters:
+* `Name`
+* `Template` -- the name of the CloudForamtion template yaml file to use
```
Comment (kswenson) [inline scripts/README.md:19]:
```suggestion
* `Template` -- the name of the CloudFormation template yaml file to use
```

File: scripts/README.md [hunk 3]
```diff
@@ -1,4 +1,36 @@
...
+    parameter set from `stack-params/*.yml`
+* `ParameterModifications` — Values to replace from the stack-params.
+
+## .gitignore
+Because you will be creating config files that might have passwords in them,
+git will ginore `.yml` files in the `configs` and `stack-params` folders. Sample
+files have been renamed `.sample-yml` which might break editor configurations.
+
+
+reads a create-config.yml file and then duplicates
```
Comment (kswenson) [inline scripts/README.md:33]:
```suggestion
reads a `create-config.yml` file and then duplicates
```

**Comment by knowuh** [comment]
@kswenson  — Thanks, you are always catching my typos! I appreciate that.

You have prompted me to install [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) for VSCode.

I will also try to remember to proof read my markdown files.  😄

**Comment by knowuh** [comment]
@scytacki -- this PR has been rebased from #41

It could be reviewed after we convince ourselves that the template changes of #41 are not going to break deployments to production or staging learn.

**Comment by knowuh** [comment]
@scytacki Do you think this work can be merged?  Some updates to the script and documentation.

File: README.md [hunk 4]
```diff
@@ -22,6 +22,35 @@ Running script to create an app-only portal stack with parameters copied from an
...
+See the file ./scripts/README.md for more details.
+
+## Paired QA servers (Lara / Portal)
+2021-01-21: We want to be able tp deploy paired sets of Lara & Portal to the QA environment.
+Recent template changes should make this easier.
+
+1. Change into the `scripts` directory and install the dependencies: `cd scripts && npm install`
+2. Create some configuration files. These config templates will setup your stacks for you. See the `README.md` file in the scripts directory for more info: `cp configs/create-config.sample.yml configs/create-my-project-config.yml`
+3. Copy Parameters from production or staging stacks.
+	1. Select your primary AWS account credentials to copy stack parameters for the production Lara and Portal instances you are working on. `export AWS_PROFILE=concord-qa`
```
Comment (scytacki) [inline README.md:34]:
```suggestion
	1. Select your primary AWS account credentials to copy stack parameters for the production Lara and Portal instances you are working on. `export AWS_PROFILE=concord`
```

**Review by scytacki — CHANGES_REQUESTED** [review]
I found one issue in the documentation. Otherwise it looks good.

**Comment by scytacki** [comment]
It is a little weird that this PR is showing changes that are already part of master. I'm going to merge master into this branch to remove those changes from the PR.

**Review by scytacki — APPROVED** [review]
Looks great.

**Pull request merged**